### PR TITLE
Fix #198 issue.

### LINKED ISSUE #198: Gateway start failure without probe.

### PR DESCRIPTION
**Pull request by Xiangrui2019**
fix #198

### CODE REVIEW

File: src/Abstracts/XelNaga/Services/AsyncHelper.cs [hunk 1]
```diff
@@ -28,6 +28,22 @@ public static void TryAsyncThreeTimes(Func<Task> steps)
             }
         }
 
+        public static void TryAsyncForever(Func<Task> steps)
+        {
```
Comment (Anduin2017) [inline src/Abstracts/XelNaga/Services/AsyncHelper.cs:32]: I suggest that in this method, you can reuse the TryAsync method. Passing the times: int.Max.

Comment (Anduin2017) [reply src/Abstracts/XelNaga/Services/AsyncHelper.cs:32]: So the code can be simplified.

File: src/SDK/SDK/Extends.cs [hunk 2]
```diff
@@ -264,12 +264,12 @@ public static IServiceCollection AddBasic(this IServiceCollection services, para
 
         public static IServiceCollection UseBlacklistFromAddress(this IServiceCollection services, string address)
         {
-            AsyncHelper.TryAsyncThreeTimes(async () =>
+            AsyncHelper.TryAsync(async () =>
```
Comment (Anduin2017) [inline src/SDK/SDK/Extends.cs:267]: Why not use 'TryAsyncForever'? For failed loading blacklist will cause the service unavailable.

**Review by Anduin2017 — CHANGES_REQUESTED** [review]
Nice one. While I have some feedback.

File: src/SDK/SDK/Extends.cs (hunk 2 above)
Comment (Xiangrui2019) [reply src/SDK/SDK/Extends.cs:267]: OK, Thanks

File: src/Abstracts/XelNaga/Services/AsyncHelper.cs (hunk 1 above)
Comment (Xiangrui2019) [reply src/Abstracts/XelNaga/Services/AsyncHelper.cs:32]: OK, Thanks

**Review by Anduin2017 — APPROVED** [review]
Nice one!!!

**Pull request merged**